pagopa/io-app
# feat: [IOCOM-849,IOCOM-850] Add `MessageDetailsComponent` in the detail of a message

## Short description
This PR adds the `MessageDetailsComponent` in the detail of a message. If message is expired, the tag _expired_ will be displayed.

<details open><summary>Details</summary>
<p>

| with payment and due date - valid | with payment and due date - expired |
| - | - |
| <img src="https://github.com/pagopa/io-app/assets/29163287/4a62c235-1dea-483a-bf68-4ab395ef8815" width="300"/> | <img src="https://github.com/pagopa/io-app/assets/29163287/61b793d3-51e8-4ee3-946b-90ea8aac3f75" width="300"/> |

</p>
</details> 

## List of changes proposed in this pull request
- added the `MessageDetailsComponent` in the detail of a message
- added the tag _expired_
- update langs
- added tests

## How to test
Using `io-dev-api-server`, generate messages with valid/invalid `due_date`. Open the message and check that the data is displayed correctly.
